js_modules/dagster-ui/packages/ui-components/src/components/TagSelector.tsx

```diff
@@ -47,12 +48,23 @@ type Props = {
 const defaultRenderTag = (tag: string, tagProps: TagSelectorTagProps) => {
   return (
     <Tag key={tag}>
-      <Box flex={{direction: 'row', gap: 4, justifyContent: 'space-between', alignItems: 'center'}}>
-        <span>{tag}</span>
+      <div
+        style={{
+          display: 'grid',
```

Comment:
I opted to use `display grid` over adding `grow: 1` on the children components because I only have to set styles on the parent div rather than both the parent and the child divs, I also find it more reliable

Reply:
This seems fine to me! Maybe sometime soon we bake grid into `Box`